unlock-app/src/middlewares/providerMiddleware.ts

```diff
@@ -1,3 +1,4 @@
+import { setTimeout } from 'timers'
```

Comment:
oops :) looks like VS Code got too clever for you

Reply:
Good catch! Thanks